lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -151,7 +151,14 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
             }
 
             if (this.isPathVariableType(DestinationFolderPathType.FOLDER_TYPE)) {
-                rootNodeId = this.field.params.fileSource.destinationFolderPath.value;
```

Comment:
This method does too many things... can we split it into a more meaningful method?